app/marketing/tests/test_marketing_utils.py

```diff
@@ -0,0 +1,92 @@
+# -*- coding: utf-8 -*-
+"""Handle gas util related tests.
+
+Copyright (C) 2018 Gitcoin Core
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU Affero General Public License as published
+by the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+GNU Affero General Public License for more details.
+
+You should have received a copy of the GNU Affero General Public License
+along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+"""
+from django.test import TestCase
+
+from marketing.models import EmailSubscriber, Stat
+from marketing.utils import get_or_save_email_subscriber, get_stat, should_suppress_notification_email
+
+
+class MarketingStatUtilsTest(TestCase):
+    """Define tests for marketing utils."""
+
+    def setUp(self):
+        """Perform setup for the testcase."""
+        Stat.objects.create(
+            key='mykey',
+            val=1
+        )
+        Stat.objects.create(
+            key='mykey',
+            val=2
+        )
+
+    def test_get_stat(self):
+        """Test the marketing util get_stat method."""
+        assert get_stat('mykey') == 2
+
```

Comment:
Can you add another new line between these?

Reply:
you want to separate the classes by 2 lines, right?

Reply:
Yes @leonprou - Two new lines between module level classes and methods.

```python
import thing


class Example:
    """Some docstring."""

    def test(self):
        """Some test."""
        return 'test'

    def test_again(self):
        """Some test again."""
        return 'test again'


class ExampleAgain:
    """Some docstring again."""

    def test(self):
        """Some test."""
        return 'test'


def some_test():
    """Some test."""
    return 'some test'

```